napari/components/dims.py

```diff
@@ -194,6 +194,10 @@ def set_range(
         _range: Union[
             Sequence[Union[int, float]], Sequence[Sequence[Union[int, float]]]
         ],
+        *,
+        restore_point: Union[
+            Union[int, float], Sequence[Union[int, float]]
+        ] = None,
```

Comment:
```suggestion
        *,
        restore_point: Optional[Sequence[float]] = None,
```
Based on docstring and code call

Reply:
Changed :)